File: phpKraken/Customer.php
<?php 
	$con=mysqli_connect("mysql14.000webhost.com","a4107973_filip","IsakAsimov1943","a4107973_filip");


	

	$customerName = $_POST["customerName"];
	$customerSurname = $_POST["customerSurname"];
	$customerAddress = $_POST["customerAddress"];
	$customerPhone = $_POST["customerPhone"];
	$customerRegular = $_POST["customerRegular"];
	



	$statement = mysqli_prepare($con, "INSERT INTO Customer(customer_name, customer_surname, customer_address, customer_phone,customer_regular) VALUES (?,?,?,?,?)");
	mysqli_stmt_bind_param($statement,"ssssi", $customerName, $customerSurname, $customerAddress, $customerPhone,$customerRegular);
	mysqli_stmt_execute($statement);

	mysqli_stmt_close($statement);



	mysqli_close($con);


?>

File: app/src/main/java/filipivanov/kraken/Customer.java
package filipivanov.kraken;

import java.io.Serializable;

/**
 * Created by WakingBliss on 9/15/2015.
 */
public class Customer implements Serializable {

    String  customerName,customerSurname,customerAddress,customerPhone;


    public Customer(String customerName, String customerSurname, String customerAddress, String customerPhone   ){

        this.customerName = customerName;
        this.customerSurname = customerSurname;
        this.customerAddress = customerAddress;
        this.customerPhone = customerPhone;


    }
}
